src/server/src/main/docker/Dockerfile

```diff
@@ -35,21 +37,35 @@ ENV REAPER_SEGMENT_COUNT=200 \
     REAPER_CASS_AUTH_ENABLED="false" \
     REAPER_CASS_AUTH_USERNAME="cassandra" \
     REAPER_CASS_AUTH_PASSWORD="cassandra" \
+    REAPER_CASS_NATIVE_PROTOCOL_SSL_ENCRYPTION_ENABLED="false" \
     REAPER_DB_DRIVER_CLASS="org.h2.Driver" \
     REAPER_DB_URL="jdbc:h2:/var/lib/cassandra-reaper/db;MODE=PostgreSQL" \
     REAPER_DB_USERNAME="" \
     REAPER_DB_PASSWORD="" \
+    REAPER_METRICS_ENABLED=false \
+    REAPER_METRICS_FREQUENCY="1 minute" \
+    REAPER_METRICS_REPORTERS="[]" \
     JAVA_OPTS=""
 
 ADD cassandra-reaper.yml /etc/cassandra-reaper.yml
 ADD entrypoint.sh /usr/local/bin/entrypoint.sh
-ADD append-persistence.sh /usr/local/bin/append-persistence.sh
+ADD configure-persistence.sh /usr/local/bin/configure-persistence.sh
```

Comment:
I like `configure` a lot better. Nice job fixing that.

Reply:
No worries. Thanks.